packages/react-composites/src/composites/CallComposite/utils/MediaGalleryUtils.ts

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { _formatString } from '@internal/acs-ui-common';
+import { RemoteParticipantState } from '@internal/calling-stateful-client';
+import { useEffect, useMemo, useState } from 'react';
+import { useLocale } from '../../localization';
+import { CallAdapterState } from '../adapter';
+import { useSelector } from '../hooks/useSelector';
+
+const getRemoteParticipantsConnectedSelector = (callState: CallAdapterState): RemoteParticipantState[] => {
```

Comment:
if its a selector lets keep with all the other selectors in the other packlet

Reply:
Since this is a custom selector explicitly for this hook should we be storing it with the mediaGallery selectors?

Reply:
I think media gallery selector works, just as long as its in thie folder with the oder selectors
![image](https://user-images.githubusercontent.com/2684369/202592514-d43912bb-8ad3-4935-b0ea-dce5172060b6.png)